rust-app/src/settings.rs

```diff
@@ -0,0 +1,70 @@
+#[cfg(target_os = "nanos")]
```

Comment:
This will make everything nanos-only
```suggestion
#![cfg(target_os = "nanos")]

```
would that be easier?

Reply:
hmm we have non-nanos stuff below

Reply:
I would do
```rust
#[cfg(target_os = "nanos")]
mod nano {
  // ...
}
#[cfg(target_os = "nanos")]
use nanos::*;
```
then